managed/src/main/java/com/yugabyte/yw/scheduler/Scheduler.java

```diff
@@ -251,6 +254,17 @@ private void runMultiTableBackupsTask(Schedule schedule) {
   }
 
   private void runDeleteBackupTask(Customer customer, Backup backup) {
+    UUID universeUUID = backup.getBackupInfo().universeUUID;
+    try {
+      Universe universe = Universe.get(universeUUID);
+      if (universe.getUniverseDetails().universePaused) {
+        LOG.warn("Cannot delete the backup since the universe {} is currently in a paused state.",
+            universeUUID.toString());
+        return;
+      }
+    } catch (Exception e) {
+      LOG.warn("Universe not found for the backup {}.", backup.backupUUID.toString());
```

Comment:
"Universe {} not found for the backup {}." - is better as we don't need to search universe uuid inside the backup configuration.